src/main/webapp/i18n/de/courseTutorialGroups.json

```diff
@@ -0,0 +1,29 @@
+{
+    "artemisApp": {
+        "breadcrumb": {
+            "title": "Übungsgruppen"
+        },
+        "pages": {
+            "courseTutorialGroups": {
+                "title": "Deine angemeldeten Übungsgruppen",
+                "noRegistrations": "Du bist in keiner Übungsgruppe angemeldet.",
+                "allFilter": "Alle Übungsgruppen anzeigen",
+                "registeredFilter": "Angemeldete Übungsgruppen anzeigen"
+            },
+            "courseTutorialGroupDetail": {
+                "title": "Übungsgruppen Details"
+            },
+            "courseTutorialGroupOverview": {
+                "title": "Übungsgruppen Übersicht",
```

Comment:
Either hyphenate or make one word or "Übersicht der Übungsgruppen"

Reply:
Done in 1a8ae32df3469f96ada97d6dfa074467d675a5ec